resources/idlharness.js

```diff
@@ -1085,6 +1085,83 @@ IdlInterface.prototype.test_self = function()
         }
     }.bind(this), this.name + " interface: existence and properties of interface prototype object");
 
+    if (this.is_global() && typeof Object.setPrototypeOf === "function") {
+        // These functions test WebIDL as of 2017-05-29.
+        // https://heycam.github.io/webidl/#named-properties-object-setprototypeof
+        test(function() {
+            var originalValue = Object.getPrototypeOf(self[this.name].prototype);
+            var newValue = Object.create(null);
+
+            assert_throws(new TypeError(), function() {
+                Object.setPrototypeOf(self[this.name].prototype, newValue);
+            });
+
+            assert_equals(
+                    Object.getPrototypeOf(self[this.name].prototype),
+                    originalValue,
+                    "original value not modified"
+                );
+        }.bind(this), this.name + " interface: internal [[SetPrototypeOf]] method " +
+            "of named properties object - setting to a new value via Object.setPrototypeOf " +
```

Comment:
No... I thought you were referencing https://heycam.github.io/webidl/#platform-object-setprototypeof